### PR TITLE
[9096] Added user setting to change Handbook url in Help > Manual 

### PR DESCRIPTION
**Pull request by jamda**

### CODE REVIEW

**Comment by col-panic** [comment]
In Files changed I can not see the changes made, as entire files are being shown. Please prepare the patch such that the changes can be easily seen. I guess this is due to some formatting problem, please be sure to use the ElexisFormatter

**Comment by jamda** [comment]
I don't understand i commited the same way as before. And am using the .xml file for formatting.

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 1]
```diff
@@ -1,961 +1,961 @@
...
+			public void run() {
+				Desktop desktop = Desktop.getDesktop();
+				String url = CoreHub.globalCfg.get(UiPreferenceConstants.CFG_HANDBOOK,
+						UiPreferenceConstants.DEFAULT_HANDBOOK);
+				if (Desktop.isDesktopSupported()) {
+					try {
+						desktop.browse(new java.net.URI(url));
+					} catch (Exception e) {
+						logger.warn("failed to open default browser :" + e);
+						MessageDialog.openError(mainWindow.getShell(), Messages.GlobalActions_Error,
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: Added an error message if user doesn't put a valid URL in Préférences > Iatrix Wiki Hilfe

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java [hunk 2]
```diff
@@ -1,961 +1,961 @@
...
+			{
+				setImageDescriptor(Images.IMG_BOOK.getImageDescriptor());
+				setToolTipText(Messages.GlobalActions_ac_openhandbook); // $NON-NLS-1$
+
+			}
+
+			@Override
+			public void run() {
+				Desktop desktop = Desktop.getDesktop();
+				String url = CoreHub.globalCfg.get(UiPreferenceConstants.CFG_HANDBOOK,
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/GlobalActions.java]: Changed the url variable to be set in Iatrix Hilfe préférences.

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/Messages.java [hunk 3]
```diff
@@ -1,106 +1,101 @@
...
+	public static String GlobalActions_DeleteKons;
+	public static String GlobalActions_EditCase;
+	public static String GlobalActions_Error;
+	public static String GlobalActions_Home;
+	public static String GlobalActions_HomeToolTip;
+	public static String GlobalActions_Import;
+	public static String GlobalActions_ImportDlgMessage;
+	public static String GlobalActions_ImportDlgShelltext;
+	public static String GlobalActions_ImportDlgTitle;
+	public static String GlobalActions_IatrixPreferences_URL;
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/Messages.java]: The error message variable.

**Comment by col-panic** [comment]
Humm, this is really a problem. Could you start from fresh ground resetting to the current state and then applying your patches? I simply can't see the diff (take a look at GlobalActions for example) which really is a problem for me to accept the patch!

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/messages.properties [hunk 4]
```diff
@@ -1,85 +1,85 @@
...
+GlobalActions_CouldntDeleteKons            = could not delete consultation
+GlobalActions_CouldntDeleteKonsExplanation = only a consultation without diagnoses and services can
+GlobalActions_Cut                          = &Cut
+GlobalActions_DeleteCase                   = delete case
+GlobalActions_DeleteKons                   = delete consultation
+GlobalActions_EditCase                     = edit case
+GlobalActions_Error                        = error
+GlobalActions_Home                         = Home
+GlobalActions_HomeToolTip                  = return to the initial perspective
+GlobalActions_IatrixPreferences_URL        = Enter a URL of type https://wiki.elexis.info in parameteres > Iatrix Wiki Hilfe > Handbuch
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/messages.properties]: The error message.

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/messages_de.properties [hunk 5]
```diff
@@ -1,86 +1,85 @@
...
+GlobalActions_CouldntDeleteKons            = Konsultation konnte nicht gel\u00F6scht werden
+GlobalActions_CouldntDeleteKonsExplanation = Eine Konsultation kann nur dann gel\u00F6scht werden, wenn sie weder\n
+GlobalActions_Cut                          = &Ausschneiden
+GlobalActions_DeleteCase                   = Fall l\u00F6schen
+GlobalActions_DeleteKons                   = Konsultation l\u00F6schen
+GlobalActions_EditCase                     = Fall bearbeiten
+GlobalActions_Error                        = Fehler
+GlobalActions_Home                         = Anfang
+GlobalActions_HomeToolTip                  = Zur Startperspektive zur\u00FCckkehren
+GlobalActions_IatrixPreferences_URL        = Bitte w\u00E4hlen Sie ein URL von typ https://wiki.elexis.info in Einstellungen > Itrix Wiki Hilfe > Handbuch
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/messages_de.properties]: The same error message in German

File: ch.elexis.core.ui/src/ch/elexis/core/ui/actions/messages_fr.properties [hunk 6]
```diff
@@ -1,84 +1,84 @@
...
+GlobalActions_CouldntDeleteKons            = la consultation ne pouvait pas \u00EAtre effac\u00E9e
+GlobalActions_CouldntDeleteKonsExplanation = une consultation ne peut \u00EAtre effac\u00E9e que , lorsqu'elle ne contient ni diagnostics ni services
+GlobalActions_Cut                          = &couper
+GlobalActions_DeleteCase                   = effacer le cas
+GlobalActions_DeleteKons                   = effacer la consultation
+GlobalActions_EditCase                     = traiter le cas
+GlobalActions_Error                        = erreur
+GlobalActions_Home                         = page d'accueil
+GlobalActions_HomeToolTip                  = retour \u00E0 la perspective initiale
+GlobalActions_IatrixPreferences_URL        = entrez un URL de type https://wiki.elexis.info dans param\u00E8tres > Iatrix Wiki Hilfe > Handbuch
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/actions/messages_fr.properties]: The same error message in French

File: ch.elexis.core.ui/src/ch/elexis/core/ui/constants/UiPreferenceConstants.java [hunk 7]
```diff
@@ -1,35 +1,34 @@
...
+
+	public static final String USERSETTINGS2_EXPANDABLE_COMPOSITES_BASE = "view/expandableComposites";
+	public static final String USERSETTINGS2_EXPANDABLE_COMPOSITES = USERSETTINGS2_EXPANDABLE_COMPOSITES_BASE
+			+ "/setting";
+	public static final String USERSETTINGS2_EXPANDABLE_COMPOSITES_STATES = USERSETTINGS2_EXPANDABLE_COMPOSITES_BASE
+			+ "/states/";
+	public static final String USERSETTINGS2_EXPANDABLECOMPOSITE_STATE_OPEN = "1";
+	public static final String USERSETTINGS2_EXPANDABLECOMPOSITE_STATE_CLOSED = "2";
+	public static final String USERSETTINGS2_EXPANDABLECOMPOSITE_STATE_REMEMBER_STATE = "3";
+	public static final String DEFAULT_BASE_URL = "http://wiki.elexis.info/";
```
Comment (jamda) [inline ch.elexis.core.ui/src/ch/elexis/core/ui/constants/UiPreferenceConstants.java]: Line 29 to 33 refactored a few variables so they are set in this file

**Pull request closed without merging**